components/BackupAndRestoreBannerNotification.tsx

```diff
@@ -54,7 +54,9 @@ export const BackupAndRestoreBannerNotification: React.FC = () => {
         />
       )}
 
-      {backUpController.isBackingUpFailure && backupFailure()}
+      {backUpController.isBackingUpFailure &&
+        !backUpController.isFetchDataFromDB &&
```

Comment:
We are in success / failure state only when backup is success / failure respectively, Why do we require this additional check of if its not fetching from DB?

Once user initiated backup, 
- we move to backingUp state's initial state - backingUp.checkDataAvailability 
- and after backup is success we move to backingUp.success state. 
- If user again initiates backup, we land in backingUp.checkDataAvailability state. So here we are not in backingUp.success state, right? If this is the case why do we need extra check of !backUpController.isFetchDataFromDB? 

Please let me know if I am getting it right here!